.env_example

```diff
@@ -1,23 +0,0 @@
-# Username and password to log in to a docassemble playground to run tests
```

Comment:
I understand that this file doesn't need be a file on the filesystem anymore for the tests to run (since everything is in Github now), but isn't it still a useful piece of documentation? PLAYGROUND_EMAIL,  PLAYGROUND_PASSWORD, PLAYGROUND_ID, and EXTRA_LANGUAGES are still needed, right?

Reply:
They are needed (until v4 comes out, and then it's something else), but the setup interview creates them. I feel that if we want to support this information, can add it to the documentation. It doesn't feel like it belongs as as a separate file in every repo. It's there for local tests and, if we're keeping support for that, we'd need the package.json too, etc. Thoughts?

Reply:
That's fair, it is better for it to be in the documentation. +1 for keeping it removed then.